scripts/jenkins/Makefile.jenkins

```diff
@@ -485,3 +485,11 @@ check-leaks:
 		echo "Credentials leaked! Please check $(PWD)/leak-check.out"; \
 		exit 1; \
 	fi
+
+h2o-docker-build:
+    docker build -t h2oai/h2o-open-source:$(DOCKER_IMAGE_TAG) -f docker/public/Dockerfile-h2o-release .
```

Comment:
I think it might be a good idea to denote that this image is meant for k8s deployments. WDYT?

Reply:
Agreed. Doing it now.

Reply:
Done.